core/domain/stats_services.py

```diff
@@ -527,10 +647,15 @@ def get_playthrough_by_id(playthrough_id):
     """
     playthrough_model = (
         stats_models.PlaythroughModel.get(playthrough_id, strict=False))
-    return playthrough_model and get_playthrough_from_model(playthrough_model)
+    if playthrough_model is None:
+        return None
+
+    return get_playthrough_from_model(playthrough_model)
 
 
-def get_exploration_stats_by_id(exp_id, exp_version):
+def get_exploration_stats_by_id(
+    exp_id: str, exp_version: int
+) -> Optional[stats_domain.ExplorationStats]:
```

Comment:
Can you update the docstring too? It does not address the None case.

Reply:
Docstring Updated!